### PR TITLE
Minor updates to jibe with NOAO -> NOIR for their datalab

### PR DESCRIPTION
**Pull request by profxj**
Also added a test for building hosts

### CODE REVIEW

**Comment by profxj** [comment]
frb/tests/test_associate.py F                                           [  1%]
frb/tests/test_build.py .                                               [  3%]
frb/tests/test_eazy.py .                                                [  5%]
frb/tests/test_experiment.py .                                          [  6%]
frb/tests/test_finder.py .                                              [  8%]
frb/tests/test_frb.py ..                                                [ 11%]
frb/tests/test_frbdlas.py ...                                           [ 16%]
frb/tests/test_frbem.py ..                                              [ 20%]
frb/tests/test_frbgalaxies.py .....                                     [ 28%]
frb/tests/test_frbhalos.py ..........                                   [ 45%]
frb/tests/test_frbigm.py .....                                          [ 53%]
frb/tests/test_frbio.py .                                               [ 55%]
frb/tests/test_frbobs.py .                                              [ 56%]
frb/tests/test_frbrm.py .                                               [ 58%]
frb/tests/test_frbsurveys.py ........                                   [ 71%]
frb/tests/test_frbtscatt.py ...                                         [ 76%]
frb/tests/test_galaxies.py .                                            [ 78%]
frb/tests/test_galfit.py .s.                                            [ 83%]
frb/tests/test_mcmc.py ...                                              [ 88%]
frb/tests/test_offset.py .                                              [ 90%]
frb/tests/test_photom.py ...                                            [ 95%]
frb/tests/test_scripts.py ...                                           [100%]

The `associate` test is fixed in the other PR

File: frb/surveys/des.py [hunk 1]
```diff
@@ -13,8 +16,7 @@
     print("Warning:  You need to install pyvo to retrieve DES images")
     _svc = None
 else:
-    _DEF_ACCESS_URL = "https://datalab.noao.edu/sia/des_dr2"
-    _svc = sia.SIAService(_DEF_ACCESS_URL)
+    _svc = sia.SIAService(defs.NOIR_DEF_ACCESS_URL)
```
Comment (SunilSimha) [inline frb/surveys/des.py:19]: DES should not have the same access URL as DECaLs which is what `NOIR_DEF_ACCESS_URL` is currently defined as. Every survey has a different URL.

File: frb/surveys/defs.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# NOIR Lab definitions
+NOIR_DEF_ACCESS_URL = "https://datalab.noirlab.edu/sia/ls_dr8"
```
Comment (SunilSimha) [inline frb/surveys/defs.py:2]: Needs one for each survey.

File: frb/surveys/wise.py [hunk 3]
```diff
@@ -15,8 +18,7 @@
     print("Warning: You need to install pyvo to use the survey tools...")
     _svc = None
 else:
-    _DEF_ACCESS_URL = "https://irsa.ipac.caltech.edu/ibe/data/wise/allwise/p3am_cdd"
-    _svc = sia.SIAService(_DEF_ACCESS_URL)
+    _svc = sia.SIAService(defs.NOIR_DEF_ACCESS_URL)
```
Comment (SunilSimha) [inline frb/surveys/wise.py:21]: And this one too.

**Review by SunilSimha — CHANGES_REQUESTED** [review]
Access URLs are survey-specific and I guess the tests are still in progress but otherwise, it looks okay.

File: frb/surveys/des.py (hunk 1 above)
Comment (profxj) [reply frb/surveys/des.py:19]: Ah, got it.  My bad.  Fixing..

File: frb/surveys/wise.py (hunk 3 above)
Comment (profxj) [reply frb/surveys/wise.py:21]: fixed

File: frb/surveys/defs.py (hunk 2 above)
Comment (profxj) [reply frb/surveys/defs.py:2]: fixed

File: frb/surveys/wise.py (hunk 3 above)
Comment (profxj) [reply frb/surveys/wise.py:21]: fixed

**Review by SunilSimha — APPROVED** [review]
Looks good to merge!

**Pull request merged**